pkg/scheduler/queue/user_queues.go

```diff
@@ -103,6 +105,9 @@ func (q *queues) len() int {
 }
 
 func (q *queues) deleteQueue(userID string) {
+	q.userQueuesMx.Lock()
```

Comment:
this line is not covered by the unit test

Reply:
modified the test to cover this line